src/Data/MonadicStreamFunction/InternalCore.hs

```diff
@@ -133,3 +132,66 @@ reactimate :: Monad m => MSF m () () -> m ()
 reactimate sf = do
   (_, sf') <- unMSF sf ()
   reactimate sf'
+
+-- Implementation of some common operations for type-class instances, without the instances
+map_msf :: Functor m => (b -> c) -> MSF m a b -> MSF m a c
```

Comment:
Underscores in names are not OK.

Reply:
This should rather be a `Functor` instance for `MSF`s, and not a standalone function.

Reply:
I thought of that, but I did not want to add any type-class instances, since they impact modularity. A function can be imported manually.